docs/user/integrations.rst

```diff
@@ -1,226 +1,19 @@
-VCS Integrations
-================
+Connecting git repositories
+===========================
+
+Read the Docs is a *Continuous Integration* platform for your git repository.
```

Comment:
```suggestion
Read the Docs is a *Continuous Documentation Deployment* platform for your software project.
```

I think this is closer to what we want to say.

Reply:
Sounds good :+1: The word "Continuous" is helpful in case the reader is already familiar with other build services like Travis, Jenkins, Circle.. even GitBook.. they might recognize the setup.